packages/workspace-store/src/schemas/v3.1/strict/parameter.ts

```diff
@@ -41,6 +47,10 @@ type ParameterObjectBase = {
   deprecated?: boolean
   /** If true, clients MAY pass a zero-length string value in place of parameters that would otherwise be omitted entirely, which the server SHOULD interpret as the parameter being unused. Default value is false. If style is used, and if behavior is n/a (cannot be serialized), the value of allowEmptyValue SHALL be ignored. Interactions between this field and the parameter's Schema Object are implementation-defined. This field is valid only for query parameters. Use of this field is NOT RECOMMENDED, and it is likely to be removed in a later revision. */
   allowEmptyValue?: boolean
+  /** An OpenAPI extension to mark parameters as read-only */
+  'x-is-readonly'?: boolean
+  /** An OpenAPI extension to mark parameters as global */
+  'x-is-global'?: boolean
```

Comment:
soo the OpenAPI-way to name booleans would be just `x-readonly` and `x-disabled` (see `deprecated` in the Operation Object)

so it looks a bit foreign to me, looking at it with OpenAPI-eyes

But I leave it up to you

Reply:
sounds reasonable